src/builder/import.rs

```diff
@@ -17,11 +17,17 @@ impl ImportBuilder {
 	}
 }
 
+impl Default for ImportBuilder {
+	fn default() -> Self {
+		Self::new()
+	}
+}
+
```

Comment:
Why all those extra `Default` implementations? Is it used somewhere or is is for the create users?

Reply:
This is for the crate users who are to be expected to look for a `Default` impl. if fn new()` takes no input. See https://rust-lang.github.io/rust-clippy/master/#new_without_default